### DISCUSSION TITLE
getting "Your PC/Device needs to be repaired" during the first boot after the Windows 10 installer has rebooted

### DISCUSSION THREAD

**Discussion by parmentelat** [utmapp/UTM#5638] [Windows]
**BEFORE SUBMITTING YOUR ISSUE, PLEASE LOOK AT THE PINNED ISSUES AND USE THE SEARCH FUNCTION TO MAKE SURE IT IS NOT ALREADY REPORTED. ALWAYS COMMENT ON AN EXISTING ISSUE INSTEAD OF MAKING A NEW ONE.**

**Describe the issue**  
I am targetting windows 10 and have tried the instructions from https://docs.getutm.app/guides/windows/ 

once the windows installer has finished and reboots the machine, I am getting the following screen

![image](https://github.com/utmapp/docs.getutm.app/assets/4519227/e828f81a-98bb-49f9-b0f8-d1de832eb122)

I have tried the process with 2 different Windows 10 images

* one obtained with CrystalQuest
* one obtained here https://www.microsoft.com/fr-fr/software-download/

and I am surprised to be getting the same symptom in both cases (down to the file name and error code)

this is on a macbook pro Intel, and for example the downloaded crystalquest image is named
19045.2965.230505-1139.22h2_release_svc_refresh_CLIENTCONSUMER_RET_x64FRE_en-us.iso

I feel like I'm doing something wrong, but can't seem to figure out what exactly..

**Configuration**  
* UTM Version: 4.3.5 (87)
* macOS Version: 12.6.7
* Mac Chip (Intel, M1, ...): Intel

**Debug log**  
[UTM-debug.log](https://github.com/utmapp/UTM/files/12447692/UTM-debug.log)

**Note**

Initially I erroneously filed this as https://github.com/utmapp/docs.getutm.app/issues/28, which I closed afterwards

**Comment by paulrouget** [comment]
Exactly the same issue here, but with a M2 mac chip.
And in my case I'm not sure the installation process goes all the way to the end.

Also, installing Windows 10 x86_64.

**Comment by osy** [comment]
I think this is the same issue as https://github.com/utmapp/UTM/issues/5623 (seems like virtio driver signatures have expired). Try unmounting the spice tools iso before running the setup.

**Comment by parmentelat** [comment]
I made 2 experiments

- restarting from scratch, and **NOT** inserting spice: the problem is fixed
- restarting from scratch, inserting spice: the issue above arises
  for the record if at that time (install has gone through) I remove the spice drive, then the problem still arises
  so it looks like it's the install program that, because the spice is inserted, causes the problem

----

now...

being new to UTM, I was thinking of the spice thing in much the same way as virtualbox's guest additions, in the sense that it includes glue that makes the guest more tightly integrated into the UTM runtime
so my questions:
- is that a correct description ?
- and if so, is there a way that I can still benefit from this ? in other words is it safe to insert the spice later on down the road and install its goodies in the guest ?

thanks in advance for your insights

**Reply by osy** [reply]
The tools are fine; the signature for drivers has expired. We will need to update the drivers in a future version.

**Comment by junhochoi** [comment]
I have M1 Macbook Pro + UTM and I am trying Windows 10/11 x64 (not arm64). I also met the same error and the following workaround works for me:

- When setting up VM, follow the official procedure in the website
- Remove "Network" from the VM settings before booting CD/DVD.
- When you meet vioscsi.sys error as above, shut down VM and edit VM and disable "TPM 2.0 Device" from QEMU menu. And start the VM again and it should continue to install.
- During the installation it reboots (normal) and can get stuck. Powering off VM and running again works for me.
- Install SPICE tools (note that still no network)
- Shut down Windows and power off VM.
- "Edit" VM and add Network back. It seems "Shared Network" only works (if you need public Internet connection)
- Starting VM

Now I can browse the web using Edge, but it seems unstable - after browsing some sites, I got "STATUS_ACCESS_VIOLATION" in edge. Reloading the site usually fixes it, but ideally it shouldn't be such error...

**Comment by gk4** [comment]
It has been a month.   It appears we still have the same old spice-guest-tools-0.164.4.iso file that is expired and unusable.
